backend/src/controllers/city.controller.ts

```diff
@@ -9,7 +9,9 @@ export default class CityController {
 
         const { name }: CityAttributes = req.body;
         try {
-            const city = await City.create({ name });
+            const city = await City.upsert({ name }, {
```

Comment:
why do you use upsert?
If you need create

Should you method "addCity" have logic to update some rows in db?